service/discovery/discovery.go

```diff
@@ -277,15 +277,15 @@ func (svc *Service) Start(ctx context.Context, req *discovery.StartDiscoveryRequ
 			awsClient, err := aws.NewClient()
 			if err != nil {
 				log.Errorf("Could not authenticate to AWS: %v", err)
-				return nil, status.Errorf(codes.FailedPrecondition, "could not authenticate to AWS: %v", err)
+				continue
 			}
 			discoverer = append(discoverer,
 				aws.NewAwsStorageDiscovery(awsClient, svc.csID),
 				aws.NewAwsComputeDiscovery(awsClient, svc.csID))
 		default:
 			newError := fmt.Errorf("provider %s not known", provider)
 			log.Error(newError)
-			return nil, status.Errorf(codes.InvalidArgument, "%s", newError)
+			continue
```

Comment:
Really a continue? Shouldn't the program exit here?

Reply:
The problem is that the discovery will fail even if only one of the discoverers fails. Most prominently AWS fails if no .aws config file is present. But I want to enable "all" discoverers in the Quickstart, so that whichever work, work.

Reply:
Reverted back to old state